src/Sarif.Sarifer/IBackgroundAnalysisSink.cs

```diff
@@ -25,7 +25,9 @@ public interface IBackgroundAnalysisSink
         /// <returns>
```

Comment:
should we add the new param to the docs?

Reply:
Sorry, I put it in the wrong place :-) See below. I will fix it.

---
In reply to: [528859565](https://github.com/microsoft/sarif-visualstudio-extension/pull/284#discussion_r528859565) [](ancestors = 528859565)